source/javascripts/components/GuidedOnboarding/GuidedOnboarding.tsx

```diff
@@ -85,12 +87,16 @@ export const GuidedOnboarding = ({
     isEnabled = false,
     isOpen = false,
     onTurnOff,
-    buildStatus
+    buildStatus,
+    lastWorkflowEditedDate,
+    lastRunningBuildFinishDate,
 }: GuidedOnboardingProps): JSX.Element | null => {
   const { isMobile } = useResponsive();
 
   const stepSuccessful = {
-    run_a_build: buildStatus !== BuildStatus.Running
+    run_a_build: buildStatus !== BuildStatus.Running,
+    config_workflows: (lastRunningBuildFinishDate && lastWorkflowEditedDate) 
+                        && (lastRunningBuildFinishDate > lastWorkflowEditedDate) || false,
```

Comment:
does this work the way one would expect with date objects? should something like `getTime()` be used before comparison?

Reply:
If they're proper JavaScript `Date` objects, [it works](https://stackoverflow.com/a/493018/535401).

Reply:
mind blown

Reply:
Yep it is kind of funny. 

```
So, when + receives an object (including array) for either operand, it first calls the ToPrimitive abstract operation (section 9.1) on the value, which then calls the [[DefaultValue]] algorithm (section 8.12.8) with a context hint of number.
```

This is from a [book](https://github.com/getify/You-Dont-Know-JS) I highly recommend btw.

Bonus fun fact, you can actually check this with [Date.prototype[@@toPrimitive]](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Date/@@toPrimitive). 

```
const a = new Date()
a[Symbol.toPrimitive]('number')
```

As the MDN entry states: 
JavaScript calls the [@@toPrimitive]() method to convert an object to a primitive value. You rarely need to invoke the [@@toPrimitive]() method yourself; JavaScript automatically invokes it when encountering an object where a primitive value is expected.